TestNetTest/RacyRead timeout

```
=== RUN   TestNetTest/RacyRead
coverage: 46.2% of statements
panic: test timed out after 10m0s

goroutine 446 [running]:
testing.(*M).startAlarm.func1()
	/opt/hostedtoolcache/go/1.14.12/x64/src/testing/testing.go:1514 +0x11c
created by time.goFunc
	/opt/hostedtoolcache/go/1.14.12/x64/src/time/sleep.go:168 +0x52

goroutine 1 [chan receive, 9 minutes]:
testing.(*T).Run(0xc0000ac000, 0x6575bc, 0xb, 0x661068, 0x1)
	/opt/hostedtoolcache/go/1.14.12/x64/src/testing/testing.go:1096 +0x570
testing.runTests.func1(0xc0000ac000)
	/opt/hostedtoolcache/go/1.14.12/x64/src/testing/testing.go:1339 +0xa7
testing.tRunner(0xc0000ac000, 0xc000085ce8)
	/opt/hostedtoolcache/go/1.14.12/x64/src/testing/testing.go:1050 +0x1ec
testing.runTests(0xc00000e0c0, 0x7d4260, 0x5, 0x5, 0x0)
	/opt/hostedtoolcache/go/1.14.12/x64/src/testing/testing.go:1337 +0x595
testing.(*M).Run(0xc0000aa000, 0x0)
	/opt/hostedtoolcache/go/1.14.12/x64/src/testing/testing.go:1252 +0x300
main.main()
	_testmain.go:116 +0x338

goroutine 23 [chan receive, 9 minutes]:
testing.(*T).Run(0xc000114240, 0x656a92, 0x8, 0xc000026840, 0xc000039e01)
	/opt/hostedtoolcache/go/1.14.12/x64/src/testing/testing.go:1096 +0x570
golang.org/x/net/nettest.TestConn(0xc000114240, 0x661060)
	/home/runner/go/pkg/mod/golang.org/x/net@v0.0.0-20201031054903-ff519b6c9102/nettest/conntest.go:34 +0x20b
github.com/pion/transport/test.TestNetTest(0xc000114240)
	/home/runner/work/transport/transport/test/bridge_test.go:454 +0x45
testing.tRunner(0xc000114240, 0x661068)
	/opt/hostedtoolcache/go/1.14.12/x64/src/testing/testing.go:1050 +0x1ec
created by testing.(*T).Run
	/opt/hostedtoolcache/go/1.14.12/x64/src/testing/testing.go:1095 +0x538

goroutine 50 [sleep]:
time.Sleep(0x2710)
	/opt/hostedtoolcache/go/1.14.12/x64/src/runtime/time.go:188 +0xba
github.com/pion/transport/test.(*Bridge).Process(0xc000074f70)
	/home/runner/work/transport/transport/test/bridge.go:420 +0x6c
github.com/pion/transport/test.TestNetTest.func1.1(0xc000074f70, 0xc000012460, 0xc000012550, 0xc00001f2d0)
	/home/runner/work/transport/transport/test/bridge_test.go:462 +0x3d
created by github.com/pion/transport/test.TestNetTest.func1
	/home/runner/work/transport/transport/test/bridge_test.go:460 +0x169

goroutine 33 [semacquire, 9 minutes]:
sync.runtime_Semacquire(0xc00001f2d8)
	/opt/hostedtoolcache/go/1.14.12/x64/src/runtime/sema.go:56 +0x42
sync.(*WaitGroup).Wait(0xc00001f2d0)
	/opt/hostedtoolcache/go/1.14.12/x64/src/sync/waitgroup.go:130 +0xd4
github.com/pion/transport/test.TestNetTest.func1.2()
	/home/runner/work/transport/transport/test/bridge_test.go:477 +0x8b
golang.org/x/net/nettest.timeoutWrapper.func1()
	/home/runner/go/pkg/mod/golang.org/x/net@v0.0.0-20201031054903-ff519b6c9102/nettest/conntest.go:54 +0x3e
sync.(*Once).doSlow(0xc00001f2e0, 0xc000045e70)
	/opt/hostedtoolcache/go/1.14.12/x64/src/sync/once.go:66 +0x104
sync.(*Once).Do(0xc00001f2e0, 0xc000045e70)
	/opt/hostedtoolcache/go/1.14.12/x64/src/sync/once.go:57 +0x69
golang.org/x/net/nettest.timeoutWrapper(0xc0000ac240, 0x661060, 0x661160)
	/home/runner/go/pkg/mod/golang.org/x/net@v0.0.0-20201031054903-ff519b6c9102/nettest/conntest.go:63 +0x339
golang.org/x/net/nettest.TestConn.func3(0xc0000ac240)
	/home/runner/go/pkg/mod/golang.org/x/net@v0.0.0-20201031054903-ff519b6c9102/nettest/conntest.go:34 +0x58
testing.tRunner(0xc0000ac240, 0xc000026840)
	/opt/hostedtoolcache/go/1.14.12/x64/src/testing/testing.go:1050 +0x1ec
created by testing.(*T).Run
	/opt/hostedtoolcache/go/1.14.12/x64/src/testing/testing.go:1095 +0x538
exit status 2
FAIL	github.com/pion/transport/test	600.021s
```
https://github.com/pion/transport/pull/112/checks?check_run_id=1472463921#step:5:282 (restarted)

--- a/test/bridge_test.go
+++ b/test/bridge_test.go
@@ -460,7 +460,7 @@ func TestNetTest(t *testing.T) {
 		go func() {
 			for {
 				br.Process()
-				if conn0.closed && conn1.closed {
+				if conn0.isClosed() && conn1.isClosed() {
 					wg.Done()
 					return
 				}
@@ -474,6 +474,10 @@ func TestNetTest(t *testing.T) {
 				br.clear()
 				_ = conn0.Close()
 				_ = conn1.Close()
+
+				// Tick the clock to actually close conns.
+				br.Tick()
+
 				wg.Wait()
 			}, nil
 	})

